pkg/agent/openflow/client_test.go

```diff
@@ -37,7 +37,7 @@ func installNodeFlows(ofClient Client, cacheKey string) (int, error) {
 	gwMAC, _ := net.ParseMAC("AA:BB:CC:DD:EE:FF")
 	IP, IPNet, _ := net.ParseCIDR("10.0.1.1/24")
 	peerNodeIP := net.ParseIP("192.168.1.1")
-	err := ofClient.InstallNodeFlows(hostName, gwMAC, IP, *IPNet, peerNodeIP)
+	err := ofClient.InstallNodeFlows(hostName, gwMAC, IP, *IPNet, peerNodeIP, 0)
```

Comment:
If you just want to not block original UT, you could use types.DefaultTunOFPort for parameter of `ofport`, or the case might fail.

Reply:
Seems the test does not fail with 0. But I could change to types.DefaultTunOFPort.